jenkins/pipelines/node-linter.jenkinsfile

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env groovy
+
+pipeline {
+    agent { label 'linter' }
+    parameters{
+        string(name: 'GITHUB_ORG', defaultValue: 'nodejs', description: 'The user/org of the GitHub repo')
+        string(name: 'REPO_NAME', defaultValue: 'node', description: 'The name of the repo')
+        string(name: 'GIT_REMOTE_REF', defaultValue: 'refs/heads/master', description: 'The remote portion of the Git refspec to fetch and test')
+        string(name: 'REBASE_ONTO', defaultValue: '', description: 'Optionally, rebase onto the given ref before testing. Leave blank to skip rebasing.')
+        string(name: 'POST_REBASE_SHA1_CHECK', defaultValue: '', description: 'After rebasing, check that the resulting commit sha1 matches the given one. If left blank, no check is performed.')
+        choice(name: 'GIT_ORIGIN_SCHEME', choices: "https://github.com/\ngit@github.com:", description: '')
+        string(name: 'POST_STATUS_TO_PR', defaultValue: '', description: 'Posts build status updates to a nodejs/node PR.')
+        string(name: 'CONFIG_FLAGS', defaultValue: '', description: 'Add arguments to ./configure.')
+    }
+
+    stages {
+        stage("Setup repository") {
+            steps {
+                checkout(poll: false, scm: [
+                    $class: 'GitSCM',
+                    branches: [[
+                        name: 'refs/heads/_jenkins_local_branch'
+                    ]],
+                    userRemoteConfigs: [[
+                        url: "${params.GIT_ORIGIN_SCHEME}${params.GITHUB_ORG}/${params.REPO_NAME}",
+                        refspec: "+refs/heads/*:refs/remotes/origin/* +${params.GIT_REMOTE_REF}:refs/remotes/origin/_jenkins_local_branch"
+                    ]]
+                ])
+            }
+        }
+
+        stage('Preflight') {
+            steps {
+                sh "curl https://raw.githubusercontent.com/nodejs/build/master/jenkins/scripts/node-test-commit-pre.sh -s | bash -xe"
+                sendBuildStatus("pending", env)
+            }
+        }
+
+        stage('Run tests') {
+            steps {
+                checkMake()
+                checkSed()
+                sh """
+                  # this job does not build node, so we symlink the system's node
+                  which node #&& ln -s ${sh(script: "which node", returnStdout: true).trim()}
+                  node --version
+
+                  ${env.MAKE} lint-md-build || true
+                  # If lint-ci fails, print all the interesting lines to the cons::e.
```

Comment:
Should this be `console`?

Reply:
Yes -- this was just a raw copy-paste :)